soh/soh/Enhancements/randomizer/3drando/location_access/locacc_hyrule_field.cpp

```diff
@@ -175,8 +175,8 @@ void AreaTable_Init_HyruleField() {
 
   areaTable[RR_LON_LON_RANCH] = Area("Lon Lon Ranch", "Lon Lon Ranch", RA_LON_LON_RANCH, NO_DAY_NIGHT_CYCLE, {
                   //Events
-                  EventAccess(&logic->Epona,    {[]{return logic->Epona    || (logic->CanUse(RG_EPONAS_SONG) && logic->IsAdult && logic->AtDay);}}),
-                  EventAccess(&logic->LinksCow, {[]{return logic->LinksCow || (logic->CanUse(RG_EPONAS_SONG) && logic->IsAdult && logic->AtDay);}}),
+                  EventAccess(&logic->Epona,    {[]{return logic->Epona    || (logic->ChildsWallet && logic->CanUse(RG_EPONAS_SONG) && logic->IsAdult && logic->AtDay);}}),
```

Comment:
I think this is fine, but just to verify, Skip Epona Race skips this event altogether and just sets logic->Epona right? So that you don't need a wallet to ride Epona if Skip Epona Race is on?

Reply:
From what I've seen, it looks like skip epona race isn't accounted for in logic, which is definitely a bug but not in scope of this pr.

Reply:
Well, I'm not so sure actually. Skip Epona's song didn't have any logical implications before, because the only thing you needed to summon Epona was Epona's song and Adult access, which is also the only thing you needed to do the Epona Race. Now you also need a wallet to do the race, but if you skip the race you don't need a wallet to summon epona. Should be as simple as doing `logic->ChildsWallet || randoCtx->GetOption(RSK_SKIP_EPONA_RACE)`